bucket/audacity242.json

```diff
@@ -0,0 +1,8 @@
+{
+    "version": "2.4.2",
+    "description": "Audacity is an easy-to-use, multi-track audio editor and recorder for Windows, macOS, GNU/Linux and other operating systems.",
+    "url": "https://archive.org/download/audacity-2.4.2_202205/audacity-2.4.2.zip",
+    "hash": "0c14f7c6850c93b9dacc14fe66876b8dc3397d92dbd849898783a21bad1fff55",

```

Comment:
https://github.com/ScoopInstaller/Extras/blob/a0dddff0dee46b9b668f5aec0e53ea029c17d783/bucket/audacity.json#L7

Good that the hash matches the old manifest above, but maybe we should add the binary to the [Scoop Binary archive](https://github.com/ScoopInstaller/Binary) instead?